### ISSUE TITLE
AttributeError: 'Module_six_moves_urllib_parse' object has no attribute 'urljoin'

### ISSUE THREAD

**Issue by askaliuk** [jschneier/django-storages#71]
Django==1.6
boto==2.38.0
django-storages-redux==1.2.3

```
$ ./manage.py collectstatic

You have requested to collect static files at the destination
location as specified in your settings.

This will overwrite existing files!
Are you sure you want to do this?

Type 'yes' to continue, or 'no' to cancel: yes
Traceback (most recent call last):
  File "./manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/django/core/management/__init__.py", line 399, in execute_from_command_line
    utility.execute()
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/django/core/management/__init__.py", line 392, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/django/core/management/base.py", line 242, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/django/core/management/base.py", line 285, in execute
    output = self.handle(*args, **options)
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/django/core/management/base.py", line 415, in handle
    return self.handle_noargs(**options)
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 167, in handle_noargs
    collected = self.collect()
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 111, in collect
    handler(path, prefixed_path, storage)
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 288, in copy_file
    if not self.delete_file(path, prefixed_path, source_storage):
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 214, in delete_file
    if self.storage.exists(prefixed_path):
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/storages/backends/s3boto.py", line 435, in exists
    name = self._normalize_name(self._clean_name(name))
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/storages/backends/s3boto.py", line 362, in _normalize_name
    return safe_join(self.location, name)
  File "/Users/username/.virtualenvs/projectname/lib/python2.7/site-packages/storages/backends/s3boto.py", line 62, in safe_join
    final_path = urlparse.urljoin(final_path.rstrip('/') + "/", path)
AttributeError: 'Module_six_moves_urllib_parse' object has no attribute 'urljoin'
```

**Comment by jschneier** [comment]
Hey this is because I use Django's vendored copy of `six` and said copy for 1.6 and 1.6.1 has a bug where it doesn't trace the package correctly. This is fixed in 1.6.2, is there any way you can upgrade? Considering that extended support of 1.6 has now ended I'm strongly considering saying that this project only supports version 1.7 and up.

**Comment by jschneier** [comment]
I have decided to continue to support 1.6 but only the latest minor releases of it (1.6.11). I strongly recommend you upgrade to that if you can't go to 1.7 because there are a bunch of security and bufixes which are all backwards compatible. Going to close this.
